packages/db/src/search/CollectionSearch.ts

```diff
@@ -7,20 +7,29 @@ import { BaseSearch } from './BaseSearch';
 import { DbQueryParameters, QueryEvent } from '../types/search';
 import { translatePostgresCollectionToApiCollection } from '../translate/collections';
 import { PostgresCollectionRecord } from '../types/collection';
+import { TableNames } from '../tables';
+import { BaseRecord } from '../types/base';
 
 const log = new Logger({ sender: '@cumulus/db/CollectionSearch' });
 
-/**
- * There is no need to declare an ApiCollectionRecord type since
- * CollectionRecord contains all the same fields from the api
- */
+type StatusTypes = 'running' | 'queued' | 'completed' | 'failed' | 'total';
+type IncludeStats = {
+  [Status in StatusTypes]: number
+};
+
+interface QueryCollectionRecord extends BaseRecord, PostgresCollectionRecord {
+  statuses?: Array<string>,
+}
 
 /**
  * Class to build and execute db search query for collection
  */
 export class CollectionSearch extends BaseSearch {
+  readonly includeStats: boolean | undefined;
+
   constructor(event: QueryEvent) {
```

Comment:
remove the includeStats from `queryStringParameters` to avoid any warning messages.
const { includeStats, ...queryStringParameters } = event.queryStringParameters || {};
super({ queryStringParameters }, 'collection');
this.includeStats = (includeStats === 'true');